ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-blocks/embeddedActivityBlock/embeddedActivityBlock.component.ts

```diff
@@ -60,10 +60,16 @@ export class EmbeddedActivityBlockComponent implements OnInit {
 
     public openDeleteDialog(): void {
         const config = this.modalService.getDeleteDialogConfig(this.deleteButtonRef);
-        config.data = {
-            actionCallback: this.deleteInstance.bind(this)
-        };
-        this.dialog.open(ActivityDeleteDialogComponent, config);
+
+        const dialogRef = this.dialog.open(ActivityDeleteDialogComponent, config);
+        dialogRef.afterClosed().pipe(
+            take(1)
```

Comment:
minor: we don't need "take(1)" or unsubscribe since the observable itself completes after afterClosed.

Reply:
Fixed at eff5b0d052a090d88ed398fac960f6d9c1bb653d